### ISSUE TITLE
`search` not returning full app list

### ISSUE THREAD

**Issue by nigelhiz** [mas-cli/mas#258]
Environment:
    macOS Catalina 1015.2
    mas 1.6.3 installed from brew

Install Method:
   brew install mas

Description:
Mas doesn't seem to list all the applications in App Store, for example BBEdit. Other applications are listed perfectly fine.

On the command line run:
"mas search BBEdit"

Response:
No results found
Error: No results found

Application in App Store can be seen here:
https://apps.apple.com/us/app/bbedit/id404009241

Is this an issue with the search term or the App Store itself?

**Comment by shedali** [comment]
same issue for [Drafts](https://apps.apple.com/app/id1435957248?mt=12)

**Comment by azurinspire** [comment]
Same for Twitter. `mas search Twitter` founds all kind of 3rd party Twitter clients but not the one made by Twitter.

![Screenshot 2020-03-22 at 17 22 46](https://user-images.githubusercontent.com/40248184/77254523-d5093780-6c61-11ea-9247-1c9acdee6017.png)

**Comment by levifig** [comment]
It's getting worse… 😬

![Screen Shot 2020-05-06 at 15 12 31](https://user-images.githubusercontent.com/1605/81187469-03da3380-8fac-11ea-912c-c4b39879e07e.png)

**Comment by dadchyyy** [comment]
I have the same issue.
Not all apps are listed in most cases... 🤔

```bash
~$ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.15.4
BuildVersion:   19E287
~$ mas version
1.6.3 (brew)
~$ mas search Magnet
No results found
Error: No results found
~$ mas search Spark
No results found
Error: No results found
~$ mas search Twitter
  1418401222  Friendly for Twitter   (1.6)
  1479307973  Simplified for Twitter (1.4.1)
```

**Comment by lion-man44** [comment]
me too.
I've only noticed one thing.

If you search by ID, you'll find it.
The search command just doesn't work well though.

```
$ mas search quiver
No results found
Error: No results found

$ mas info 866773894
Quiver - take better notes 3.2.7 [9.99]
By: HappenApps, Inc.
Released: Sep 30, 2019
Minimum OS: 10.10
Size: 9.6 MB
From: https://apps.apple.com/us/app/quiver-take-better-notes/id866773894?mt=12&uo=4
```

I googled the ID.

**Comment by phatblat** [comment]
Fixed in #269

**Comment by t--takai** [comment]
Thank you very much. Thanks to your help, we were able to find Spark.

```
$ mas search spark                                                                                                                                                          03:10
  1176895641  Spark - Email App by Readdle   (2.7.0)
  1059655371  Newton - Supercharged emailing (10.0.49)
  1489591003  Email - Edison Mail            (1.0.8)
  1236045954  Canary Mail                    (3.08)
  1102826436  SparkChess 14                  (14.0.3)
  1221826210  Email Signature Creator        (1.9.4)
   869249991  Mail Detective                 (1.3.1)
   600661093  Particle Playground            (1.1)
   445147910  Fireworx Stand                 (2.6)
  1239195830  One Calendar                   (2.0.7)
  1468220753  Wave Mail - Email Client       (1.1.1)
   973942134  Warp                           (4.2)
  1393735273  Cue View                       (1.2.2)
  1473692648  Desktop Clock Fashion          (1.0)
```
